platform/src/Playground.js

```diff
@@ -60,6 +60,8 @@ if (!urlParamPrivateRepo()){
     // Public repo so no need to authenticate
     initialiseActivity();
     PlaygroundUtility.hideLogin();
+} else {
+    PlaygroundUtility.showLogin();
```

Comment:
This sets visibility to `block` when it was `inline` originally. Does it matter?

Reply:
It affects the margins but and unnoticeable in this case as it's a floating box